src/pages/Home/Home.tsx

```diff
@@ -1,9 +1,23 @@
 import React from 'react';
 
+import { useNavigate } from 'react-router-dom';
+
+import { useAppDispatch } from 'store/hooks';
+import { setUser } from 'store/UserSlice';
+
 const Home: React.FC = () => {
+  const navigate = useNavigate();
+  const dispatch = useAppDispatch();
+
+  const handleLogOut = () => {
+    dispatch(setUser({ firstName: '', lastName: '', isAuth: false }));
```

Comment:
For now it's ok, but you will need more cleanings - it may be confusing for you.
I would recommend to add action like 'clearUser', which will make inside state = initialState